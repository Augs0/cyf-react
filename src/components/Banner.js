import React from 'react'

import '../stylesheets/index.scss'
//TODO: change hrefs to react-bootstrap links
export const Banner = () => (
  <div className="page-title">
    <div className="page-title-text">
      <h1>Coding School for Refugees</h1>
      <p>Learn to code and find a job</p>
      <div className="page-title-bottom-link">
        <a className="big-link-2 btn" href="students#info">
          Apply Now
        </a>
        <a className="big-link-2 btn" href="volunteers" target="_blank" rel="noopener noreferrer">
          Become a volunteer
        </a>
      </div>
    </div>
  </div>
)