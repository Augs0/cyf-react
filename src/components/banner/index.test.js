import React from 'react';
import { shallow } from 'enzyme';

import Banner from './';

describe('Banner Component', () => {
  it('renders', () => {
    expect(shallow(<Banner />)).toMatchSnapshot();
  });
});
