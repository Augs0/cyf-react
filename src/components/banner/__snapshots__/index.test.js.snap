// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner Component renders 1`] = `
<Styled(div)>
  <Styled(img)
    alt="teaching=refugees-to-code"
    src="1.jpg"
  />
  <Styled(div)>
    <h1>
      Coding School for Refugees
    </h1>
    <p>
      Learn to code and find a job
    </p>
    <div
      className="page-title-bottom-link"
    >
      <Link
        className="big-link-2 btn"
        replace={false}
        to="students"
      >
        Apply Now
      </Link>
      <Link
        className="big-link-2 btn"
        rel="noopener noreferrer"
        replace={false}
        target="_blank"
        to="volunteers"
      >
        Become a volunteer
      </Link>
    </div>
  </Styled(div)>
</Styled(div)>
`;
